### PR TITLE
Add Sentry error logging.  Note: to implement, need to add SENTRY_DSN…

### PR DESCRIPTION
**Pull request by franksmicro**
Tested using my demo Sentry account.  Even errors not specifically caught appear to be logged.  No need to log the data or sensitivedata subfields separately - they are included in the Sentry log if passed since they are part of the exception.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,18 +2,16 @@
...
 
-const startServer = async function() {
+const startServer = async function () {
   try {
     const manifest = Manifest.get('/');
-    const server = await Glue.compose(
-      manifest,
-      composeOptions
-    );
+    const server = await Glue.compose(manifest, composeOptions);
```
Comment (JKHeadley) [inline index.js:14]: @franksmicro In general, it's best to keep formatting changes in a separate PR. Keeping them separate helps the reviewer focus on the important changes (in this case, Sentry related changes). Not a big deal here since there aren't a lot of changes, but a good thing to keep in mind.

**Review by JKHeadley — APPROVED** [review]
Looks good! Thanks Frank! I'll go ahead and merge and add the corrent SENTRY_DSN to production.

**Comment by franksmicro** [comment]
I'm not sure why my editor randomly changes things like this.  I did notice
this when I did a diff before pushing.  In the future, I will merge the old
formatting back in when I see changes like this in the diffs of files I've
worked on.

Frank

**Comment by JKHeadley** [comment]
> I'm not sure why my editor randomly changes things like this. I did notice this when I did a diff before pushing. In the future, I will merge the old formatting back in when I see changes like this in the diffs of files I've worked on. Frank

@franksmicro no worries! My editor does this sometimes as well. Usually the easiest way I manage it is to review the staged changes before committing them and un-stage any changes created by the editor.
